### PR TITLE
Create workflow for automatic patch deployment

### PR DESCRIPTION
**Pull request by raguiar9080**
Create a workflow that increases the package version (patch), commits
this change to master and then creates a new tagged release with
automatic release notes based on it. This should then trigger the other
github workflow to actually publish the package.

The workflow is on an environment that requires manual approval, meaning
the action is behind an approval in github.

The purpose is to make it easier for anyone to publish a new patch
version of the library with a single click, instead of the current
process that requires checking the code, upping the version, creating
PR, merging and then creating the release.

Signed-off-by: Ruben Aguiar <r.aguiar9080@gmail.com>

### CODE REVIEW

File: .github/workflows/up-version-patch.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.CLIENT_REBUILD_TOKEN_GITHUB }}
+        with:
+          tag_name: v${{ env.VERSION_NAME }}
+          release_name: ${{ env.VERSION_NAME }}
+          body: ${{ steps.extract_release_notes.outputs.release_notes }}
+          draft: false
+          prerelease: false
```
Comment (rbruggem) [inline .github/workflows/up-version-patch.yml:45]: How does this change package.json?

Comment (raguiar9080) [reply .github/workflows/up-version-patch.yml:45]: We have a make command that does that for us in all the necessary files (3 actually). So we just call `make patch-version`

Comment (raguiar9080) [reply .github/workflows/up-version-patch.yml:45]: PS: Internally, the command calls `npm version patch` 👍

Comment (rbruggem) [reply .github/workflows/up-version-patch.yml:45]: cool

**Review by rbruggem — APPROVED** [review]
Very much needed

**Comment by raguiar9080** [comment]
Let's see if this works 🤞

**Pull request merged**